sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/multi/MultiServiceEventClient.java

```diff
@@ -102,6 +119,124 @@ public void unregistered() {
         deregisterStep.setComplete();
     }
 
+    @Override
+    public ClientStatusResponse getClientStatus() {
+        // If the entire scheduler is uninstalling and there are no clients left to uninstall, then tell upstream that
+        // we're uninstalled.
+        boolean noClients = false;
+
+        Collection<String> finishedServices = new HashSet<>();
+        Collection<AbstractScheduler> uninstalledServices = new HashSet<>();
+
+        Collection<AbstractScheduler> services = multiServiceManager.sharedLockAndGetServices();
+        LOGGER.info("Checking status of {} service{}:", services.size(), services.size() == 1 ? "" : "s");
+        try {
+            if (services.isEmpty()) {
+                // If we don't have any clients, then WE aren't ready.
```

Comment:
`s/clients/services`?

I get that AbstractScheduler == MesosEventClient but I find it a little confusing.